### DISCUSSION TITLE
How to use river for finding related products from an image dataset

### DISCUSSION THREAD

**Discussion by anishmk1993** [online-ml/river#721] [Q&A] [answered]
Hi,
I have a dataset of products that contains more than 100k product images. How can I apply river for finding related products based on these images?
In the future, more products will be added to the dataset. So I would like to train the model with newly added data instead of retraining from scratch. Can I use river for this problem?

Thanks

**Comment by MaxHalford** [comment, marked as answer]
Hello there. River is not designed to be used for image processing. It also doesn't support information retrieval methods. You'll have to use something else 🙃
